src/Calypso-SystemPlugins-ClassScripts-Queries/ClyEvaluationScript.class.st

```diff
@@ -9,5 +9,5 @@ Class {
 
 { #category : #'script detection' }
 ClyEvaluationScript class >> isImplementedByMethod: aMethod [
-	^aMethod hasPragmaNamed: #script
+	^ aMethod hasPragmaNamed: #script
```

Comment:
Do you know why this file is marked with red here?

Reply:
No idea.